addons/viewport/src/components/Panel.jsx

```diff
@@ -0,0 +1,120 @@
+/* global document */
```

Comment:
Please import any global from the `global` package.

This allows for easy mocking with jest and makes it very explicit.